test/e2e/framework/pods.go

```diff
@@ -227,10 +225,10 @@ func (c *PodClient) WaitForErrorEventOrSuccess(pod *v1.Pod) (*v1.Event, error) {
 		}
 		for _, e := range evnts.Items {
 			switch e.Reason {
-			case events.KillingContainer, events.FailedToCreateContainer, sysctl.ForbiddenReason:
+			case "Killing", "Failed", "SysctlForbidden":
```

Comment:
Thanks @tanjunchen, just a small nit.
Can we define these as constants and comment where they are from?

Reply:
thanks 
i will update it